### PR TITLE
Project section

### PR DESCRIPTION
**Pull request by shawnthompson**
create project section under about
create Accessibility Standard page
create consultation page

### CODE REVIEW

File: src/en/about/projects/accessibility-standard/consultation/privacy-policy.html [hunk 1]
```diff
@@ -0,0 +1,35 @@
...
+</section>
+<section>
+	<h2>Audio recording consent</h2>
+	<p>This live event will be recorded and presented over Zoom for the purposes of feedback evaluation and analysis. By agreeing to participate and engage in this live event, you understand that:</p>
+	<ul>
+	<li>Any comments you may make during the event will be recorded.</li>
+	<li>You must not provide any identifiable information about yourself or others.</li>
+	<li>Your responses will not be confidential due to your audio likeness in the recording.</li>
+	</ul>
+</section>
```
Comment (duboisp) [inline src/en/about/projects/accessibility-standard/consultation/privacy-policy.html:35]: Just a comment, it is good practice to end the file with an empty line.

File: src/en/about/projects/accessibility-standard/consultation/index.html [hunk 2]
```diff
@@ -0,0 +1,98 @@
...
+		<li><a href="https://www.btb.termiumplus.gc.ca/publications/accessibilite-accessibility-eng.html">Accessibility Glossary</a></li>
+		<li><a href="https://www.canada.ca/en/employment-social-development/programs/disability/arc.html">Accessibility Resource Centre - Canada.ca</a></li>
+		<li><a href="https://www.etsi.org/deliver/etsi_en/301500_301599/301549/03.02.01_60/en_301549v030201p.pdf">EU Accessibility Standard (EN 301 549)</a></li>
+	</ul>
+</section>
+<section>
+	<h2>Contact Us</h2>
+	<p>For inquiries, do not hesitate to contact&nbsp;us by sending an email to <a href="mailto:DigitalPolicy.PolitiqueNumerique@tbs-sct.gc.ca">DigitalPolicy.PolitiqueNumerique@tbs-sct.gc.ca</a></p>
+	<p>Links to the draft Standard, feedback submission form and registration for the live sessions will be visible when the consultation begins</p>
+</section>
```
Comment (duboisp) [inline src/en/about/projects/accessibility-standard/consultation/index.html:98]: Just a comment, it is good practice to end the file with an empty line.

File: src/en/about/projects/accessibility-standard/accessibility-standard.json [hunk 3]
```diff
@@ -0,0 +1,16 @@
...
+		{
+			"title": "Projects",
+			"link": "/en/about/projects/"
+		},
+		{
+			"title": "Information and Communication Technology (ICT) Accessibility Standard",
+			"link": "/en/about/projects/accessibility-standard/"
+		}
+	]
+}
```
Comment (duboisp) [inline src/en/about/projects/accessibility-standard/accessibility-standard.json:16]: Just a comment, it is good practice to end the file with an empty line.

File: src/en/about/index.html [hunk 4]
```diff
@@ -9,4 +9,8 @@
 		<h2 class="h3"><a href="./workinggroups">Working groups on accessibility within the Government of Canada</a></h2>
 		<p>There are many working groups who cover different topics on accessbility with members from different departments working together.</p>
 	</div>
+	<div class="col-md-6">
+		<h2 class="h3"><a href="./projects">Projects</a></h2>
+		<p>There are many project that cover diverse topics on accessbility.</p>
+	</div>
 </div>
```
Comment (duboisp) [inline src/en/about/index.html:16]: Just a comment, it is good practice to end the file with an empty line.

File: src/en/about/projects/index.html [hunk 5]
```diff
@@ -0,0 +1,13 @@
...
+description: There are many project that cover diverse topics on accessibility.
+isDraft: true
+---
+
+<div class="row wb-eqht">
+	<div class="col-md-6">
+		<h2 class="h3"><a href="./accessibility-standard">Information and Communication Technology (ICT) Accessibility Standard</a></h2>
+		<p>Lorem ipsum dolor sit amet consectetur adipisicing elit. Voluptatibus, repellat voluptatum dolore iure dolores, accusantium nihil alias distinctio quae similique explicabo quam magni fugiat nisi ut laudantium a modi aperiam.</p>
+	</div>
+</div>
```
Comment (duboisp) [inline src/en/about/projects/index.html:13]: Just a comment, it is good practice to end the file with an empty line.

File: src/en/about/projects/projects.json [hunk 6]
```diff
@@ -0,0 +1,12 @@
...
+		{
+			"title": "Who we are?",
+			"link": "/en/about/"
+		},
+		{
+			"title": "Projects",
+			"link": "/en/about/projects/"
+		}
+	]
+}
```
Comment (duboisp) [inline src/en/about/projects/projects.json:12]: Just a comment, it is good practice to end the file with an empty line.

File: src/fr/about/projects/accessibility-standard/accessibility-standard.json [hunk 7]
```diff
@@ -0,0 +1,16 @@
...
+		{
+			"title": "Projects",
+			"link": "/en/about/projects/"
+		},
+		{
+			"title": "Information and Communication Technology (ICT) Accessibility Standard",
+			"link": "/en/about/projects/standard/"
+		}
+	]
+}
```
Comment (duboisp) [inline src/fr/about/projects/accessibility-standard/accessibility-standard.json:16]: Just a comment, it is good practice to end the file with an empty line.

File: src/fr/about/projects/accessibility-standard/consultation/consultation.json [hunk 8]
```diff
@@ -0,0 +1,20 @@
...
+		{
+			"title": "Information and Communication Technology (ICT) Accessibility Standard",
+			"link": "/en/about/projects/standard/"
+		},
+		{
+			"title": "Public Engagement Platform for the draft Information and Communication Technology (ICT) Accessibility Standard",
+			"link": "/en/about/projects/standard/consultation/"
+		}
+	]
+}
```
Comment (duboisp) [inline src/fr/about/projects/accessibility-standard/consultation/consultation.json:20]: Just a comment, it is good practice to end the file with an empty line.

File: src/fr/about/projects/accessibility-standard/consultation/index.html [hunk 9]
```diff
@@ -0,0 +1,98 @@
...
+		<li><a href="https://www.btb.termiumplus.gc.ca/publications/accessibilite-accessibility-eng.html">Accessibility Glossary</a></li>
+		<li><a href="https://www.canada.ca/en/employment-social-development/programs/disability/arc.html">Accessibility Resource Centre - Canada.ca</a></li>
+		<li><a href="https://www.etsi.org/deliver/etsi_en/301500_301599/301549/03.02.01_60/en_301549v030201p.pdf">EU Accessibility Standard (EN 301 549)</a></li>
+	</ul>
+</section>
+<section>
+	<h2>Contact Us</h2>
+	<p>For inquiries, do not hesitate to contact&nbsp;us by sending an email to <a href="mailto:DigitalPolicy.PolitiqueNumerique@tbs-sct.gc.ca">DigitalPolicy.PolitiqueNumerique@tbs-sct.gc.ca</a></p>
+	<p>Links to the draft Standard, feedback submission form and registration for the live sessions will be visible when the consultation begins</p>
+</section>
```
Comment (duboisp) [inline src/fr/about/projects/accessibility-standard/consultation/index.html:98]: Just a comment, it is good practice to end the file with an empty line.

File: src/fr/about/projects/index.html [hunk 10]
```diff
@@ -0,0 +1,13 @@
...
+description: There are many project that cover diverse topics on accessibility.
+isDraft: true
+---
+
+<div class="row wb-eqht">
+	<div class="col-md-6">
+		<h2 class="h3"><a href="./accessibility-standard">Information and Communication Technology (ICT) Accessibility Standard</a></h2>
+		<p>Lorem ipsum dolor sit amet consectetur adipisicing elit. Voluptatibus, repellat voluptatum dolore iure dolores, accusantium nihil alias distinctio quae similique explicabo quam magni fugiat nisi ut laudantium a modi aperiam.</p>
+	</div>
+</div>
```
Comment (duboisp) [inline src/fr/about/projects/index.html:13]: Just a comment, it is good practice to end the file with an empty line.

File: src/fr/about/projects/projects.json [hunk 11]
```diff
@@ -0,0 +1,12 @@
...
+		{
+			"title": "Who we are?",
+			"link": "/en/about/"
+		},
+		{
+			"title": "Projects",
+			"link": "/en/about/projects/"
+		}
+	]
+}
```
Comment (duboisp) [inline src/fr/about/projects/projects.json:12]: Just a comment, it is good practice to end the file with an empty line.

File: src/fr/about/projects/accessibility-standard/consultation/privacy-policy.html [hunk 12]
```diff
@@ -0,0 +1,18 @@
...
+<section class="alert alert-info">
+	<h2>Public Consultation on the ICT Accessibility Standard is now live!</h2>
+	<ul>
+		<li>The Government of Canada (GC) is excited to announce our <a href="../consultation">public consultation on the ICT Accessibility Standard</a> is now live! To participate, please provide your feedback by completing our <a href="#">feedback submission form <strong>(Link to come)</strong></a> or by attending one of our <a href="#">Live Sessions <strong>(Link to come)</strong></a></li>
+		<li>Before submitting your feedback, make sure to read the draft <a href="#">ICT Accessibility Standard <strong>(Link to come)</strong></a></li>
+		<li>The consultation will end on <em>Date and time. </em>Following this date, we will no longer be accepting feedback</li>
+		<li>If you have any questions or concerns about our consultation, or would like to provide your feedback in an alternative way, please email us at <a href="mailto:DigitalPolicy.PolitiqueNumerique@tbs-sct.gc.ca">PolitiqueNumerique@tbs-sct.gc.ca</a></li>
+		<li lang="fr"><a href="#" hreflang="fr">Ce contenu est aussi disponible en fran&ccedil;ais</a>.</li>
+	</ul>
+</section>
```
Comment (duboisp) [inline src/fr/about/projects/accessibility-standard/consultation/privacy-policy.html:18]: Wrong content. 

Le contenu de cette page devrais-être à propos de la politique de la vie privé

**Comment by shawnthompson** [comment]
Closing this PR, no traction on the idea of having a project section to our site right now

**Pull request closed without merging**